tests/kernel/qa_test_klp.pm

```diff
@@ -15,7 +15,17 @@ use base 'opensusebasetest';
 use testapi;
 use utils;
 use registration;
-use version_utils 'is_sle';
+use version_utils qw(is_sle is_transactional);
+use transactional 'trup_call';
+
+sub install_package {
+    my $command = $_[0];
+    if (is_transactional) {
+        trup_call('pkg in -l ' . $command);
+    } else {
+        zypper_call('in -l ' . $command);
+    }
+}
```

Comment:
as @mdoucha is suggesting, please move this to `lib/Wrappers/Packages.pm` (or something that isn't just dumping it on `lib/`

Reply:
I've made the changes, let's see if this works this way :crossed_fingers: